src/lib/server/Server.cpp

```diff
@@ -1407,6 +1411,22 @@ Server::handleSwitchToScreenEvent(const Event& event, void*)
 	}
 }
 
+void
+Server::handleToggleScreenEvent(const Event& event, void*)
+{
+  std::string current = getName(m_active);
+	ClientList::const_iterator index = m_clients.find(current);
+	if (index == m_clients.end()) {
+		LOG((CLOG_DEBUG1 "screen \"%s\" not active", current.c_str()));
+	}
+	else {
+    ++index;
+    if (index == m_clients.end()) index = m_clients.begin();
```

Comment:
I'd prefer to always use braces for the `if` body. Right now the code is very confusing.

Reply:
done